Dockerfile.local

```diff
@@ -9,13 +9,15 @@ WORKDIR /app
 
 COPY .build/dependencies-state.json ./.build/dependencies-state.json
 COPY .env ./
-COPY Package.resolved ./
 COPY Packages ./Packages
 
+COPY Makefile ./
+RUN make install-cmark
```

Comment:
I basically make a make command out of the script that is run on travis in my other PR: https://github.com/pointfreeco/pointfreeco/pull/100/files#diff-354f30a63fb0907d4ad57269548329e3R12